src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const classForTodoInfo = cn({
+    TodoInfo: true,
+    'TodoInfo--completed': todo.completed,
+  });
+
+  return (
+    <article data-id={todo.id} className={classForTodoInfo}>
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
better to add it here
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```
and render the component only when the user exists